crates/header-translator/translation-config.toml

```diff
@@ -739,7 +746,102 @@ skipped = true
+insertData = { skipped = true }
+deleteData = { skipped = true }
+replaceData = { skipped = true }
+substringData = { skipped = true }
+[class.DOMDocument.methods]
+createProcessingInstruction = { skipped = true }
+importNode = { skipped = true }
+createElementNS = { skipped = true }
+createAttributeNS = { skipped = true }
+getElementsByTagNameNS = { skipped = true }
+createNodeIterator = { skipped = true }
+createTreeWalker = { skipped = true }
+getOverrideStyle = { skipped = true }
+createExpression = { skipped = true }
+evaluate = { skipped = true }
+getComputedStyle = { skipped = true }
+[class.DOMElement.methods]
+setAttribute = { skipped = true }
+getAttributeNS = { skipped = true }
+setAttributeNS = { skipped = true }
+removeAttributeNS = { skipped = true }
+getElementsByTagNameNS = { skipped = true }
+getAttributeNodeNS = { skipped = true }
+hasAttributeNS = { skipped = true }
+[class.DOMEvent.methods]
+initEvent = { skipped = true }
+[protocol.DOMEventTarget.methods]
+addEventListener = { skipped = true }
+removeEventListener = { skipped = true }
+[class.DOMHTMLSelectElement.methods]
+add = { skipped = true }
+[class.DOMImplementation.methods]
+hasFeature = { skipped = true }
+createDocumentType = { skipped = true }
+createDocument = { skipped = true }
+createCSSStyleSheet = { skipped = true }
+[class.DOMMouseEvent.methods]
+initMouseEvent = { skipped = true }
+[class.DOMMutationEvent.methods]
+initMutationEvent = { skipped = true }
+[class.DOMNamedNodeMap.methods]
+getNamedItemNS = { skipped = true }
+removeNamedItemNS = { skipped = true }
+[class.DOMNode.methods]
+insertBefore = { skipped = true }
+replaceChild = { skipped = true }
+isSupported = { skipped = true }
+[class.DOMRange.methods]
+setStart = { skipped = true }
+setEnd = { skipped = true }
+compareBoundaryPoints = { skipped = true }
+[class.DOMUIEvent.methods]
+initUIEvent = { skipped = true }
+[class.DOMXPathExpression.methods]
+evaluate = { skipped = true }
+
+[class.ASAuthorizationProviderExtensionLoginConfiguration.methods]
+setCustomAssertionRequestBodyClaims_returningError = { skipped = true }
+setCustomAssertionRequestHeaderClaims_returningError = { skipped = true }
+setCustomLoginRequestBodyClaims_returningError = { skipped = true }
+setCustomLoginRequestHeaderClaims_returningError = { skipped = true }
+
+[class.ASAuthorizationProviderExtensionLoginManager.methods]
+saveLoginConfiguration_error = { skipped = true }
+
+[class.WKContentRuleListStore.methods]
+compileContentRuleListForIdentifier_encodedContentRuleList_completionHandler = { skipped = true }
+getAvailableContentRuleListIdentifiers = { skipped = true }
+lookUpContentRuleListForIdentifier_completionHandler = { skipped = true }
+removeContentRuleListForIdentifier_completionHandler = { skipped = true }
+
+[class.WebHistory.methods]
+loadFromURL_error = { skipped = true }
+saveToURL_error = { skipped = true }
+
+[protocol.WebPlugInViewFactory.methods]
+plugInViewWithArguments = { skipped = true }
```

Comment:
Could you document on each of these what the reason for their exclusion is? Since that will make it easier to include them in the future, once the issue is resolved.

Reply:
Sure, I'll add a comment for those.

I should have actually marked the PR as a draft until I can verify that the output actually works but I'll go back and double check all of these and note once it's ready.